### PR TITLE
added data import screen with support for workspace

### PR DESCRIPTION
**Pull request by ionutwho**

### CODE REVIEW

File: protected/react-src/src/services/httpMethods.js [hunk 1]
```diff
@@ -55,6 +55,28 @@ export const post = async (url, data) => {
     }
 };
 
+export const postUpload = async (url, data) => {
```
Comment (rafaelwho-en) [inline protected/react-src/src/services/httpMethods.js:61]: We need to make this compatible with Laravel Authentication. We need to add the headers Accept, Content-Type and Authorization

File: protected/react-src/src/pages/workspace/index.jsx [hunk 2]
```diff
@@ -10,35 +10,28 @@ import MuiTable from '../../components/common/table/MuiTable'
 import React from 'react'
 import replaceVariablesAsText from '../../utils/replaceVariablesAsText'
 
-const WorkspacesList = () => {
```
Comment (rafaelwho-en) [inline protected/react-src/src/pages/workspace/index.jsx:13]: Why removing this if is supposed to be the WorkspacesList file?

File: protected/react-src/src/pages/project/import.jsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import React, {useState} from "react";
```
Comment (rafaelwho-en) [inline protected/react-src/src/pages/project/import.jsx]: It is not neccesary to import React, it's imported in all the location

File: protected/react-src/src/pages/project/import.jsx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                    >
+                        <FormControlLabel value="workspace" control={<Radio />} label="Workspace" />
+                        <FormControlLabel value="hsdu_baseline" control={<Radio />} label="HSDU baseline" />
+                        <FormControlLabel value="hsdu_updates" control={<Radio />} label="HSDU updates" />
+                        <FormControlLabel value="situation_updates" control={<Radio />} label="Situation updates" />
+                    </RadioGroup>
+                </FormControl>
+            </Grid>
+            <Grid item xs={4}>
+                <Button target="_blank"
```
Comment (rafaelwho-en) [inline protected/react-src/src/pages/project/import.jsx]: This should be a Link

File: protected/react-src/src/pages/project/import.jsx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FilePresentIcon from '@mui/icons-material/FilePresent';
+import {BASE_URL} from "../../services/apiProxyService";
+
+const VisuallyHiddenInput = styled('input')({
```
Comment (rafaelwho-en) [inline protected/react-src/src/pages/project/import.jsx:19]: Baiete, maybe is better not having the css styles here.

**Review by rafaelwho-en — CHANGES_REQUESTED** [review]

File: protected/react-src/src/pages/project/import.jsx (hunk 5 above)
Comment (ionutwho) [reply protected/react-src/src/pages/project/import.jsx:19]: css in progress, will move after all data types are done

File: protected/react-src/src/pages/project/import.jsx (hunk 4 above)
Comment (ionutwho) [reply protected/react-src/src/pages/project/import.jsx]: changes as link

File: protected/react-src/src/pages/project/import.jsx (hunk 3 above)
Comment (ionutwho) [reply protected/react-src/src/pages/project/import.jsx]: removed react import

File: protected/react-src/src/pages/workspace/index.jsx (hunk 2 above)
Comment (ionutwho) [reply protected/react-src/src/pages/workspace/index.jsx:13]: used the content from Rafael

File: protected/react-src/src/services/httpMethods.js (hunk 1 above)
Comment (ionutwho) [reply protected/react-src/src/services/httpMethods.js:61]: added Authorization bearer

**Pull request merged**